### PR TITLE
update gh workflows

### PR DESCRIPTION
**Pull request by arvdias**

### CODE REVIEW

**Comment by gsanchezgavier** [comment]
looks good, could you add the Test packages action too? [sample pr](https://github.com/newrelic/nri-redis/pull/103/files)

**Comment by arvdias** [comment]
> looks good, could you add the Test packages action too? [sample pr](https://github.com/newrelic/nri-redis/pull/103/files)

Forgot we had that. Yes, i'll add those

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -135,6 +145,13 @@ jobs:
         shell: pwsh
         run: |
           build\windows\package_msi.ps1 -integration "$env:INTEGRATION" -arch ${{ matrix.goarch }} -tag "$env:TAG" -pfx_passphrase "$env:PFX_PASSPHRASE" -pfx_certificate_description "$env:PFX_CERTIFICATE_DESCRIPTION"
+      - name: Test win packages installation
+        uses: newrelic/integrations-pkg-test-action/windows@v1
+        with:
+          tag: ${{ env.TAG }}
+          integration: nri-${{ env.INTEGRATION }}
+          arch: ${{ matrix.goarch }}
+          upgrade: ${{ matrix.test-upgrade }}
```
Comment (gsanchezgavier) [inline .github/workflows/prerelease.yml:154]: needs to add the test-upgrade into the matrix

File: .github/workflows/prerelease.yml [hunk 2]
```diff
@@ -158,31 +176,34 @@ jobs:
           username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
           password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
       - name: Publish to S3 action
-        uses: newrelic/infrastructure-publish-action@main
+        uses: newrelic/infrastructure-publish-action@v1
         env:
           AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
           AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
```
Comment (roobre) [inline .github/workflows/prerelease.yml:182]: I'm not 100% sure of why these two are env vars. Couldn't them be just inputs, like the others?

**Review by roobre — APPROVED** [review]
lgtm

**Pull request merged**